kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue

```diff
@@ -166,7 +163,7 @@
         return this.wizardContext('onMyOwnOrGroup') == UsePresets.ON_MY_OWN;
       },
     },
-    mounted() {
+    created() {
```

Comment:
Curious why the change from mounted -> created?

Reply:
"Do you need to access the DOM? If no, don't use mounted." - past Richard (to me)

Reply:
Well from the look of it, we are not manipulating the DOM(at least not directly) so it justifies the change.